src/Sarif/Autogenerated/Run.cs

```diff
@@ -146,6 +146,12 @@ public SarifNodeKind SarifNodeKind
         [DataMember(Name = "defaultFileEncoding", IsRequired = false, EmitDefaultValue = false)]
         public string DefaultFileEncoding { get; set; }
 
+        /// <summary>
+        /// Specifies the unit in which the tool measures columns.
+        /// </summary>
+        [DataMember(Name = "columnKind", IsRequired = false, EmitDefaultValue = false)]
+        public string ColumnKind { get; set; }
```

Comment:
This needs to be an enum. See §3.11.19, columnKind property. Use a codegen `EnumHint`.

Reply:
Fixed.